src/services/services.ts

```diff
@@ -3839,7 +3837,9 @@ module ts {
                 if (symbolKind) {
                     pushTypePart(symbolKind);
                     displayParts.push(spacePart());
-                    addFullSymbolName(symbol);
+                    if (!symbol.isAnonymous) {
+                        addFullSymbolName(symbol);
+                    }

```

Comment:
first sorry for the delay, just recovering from a long vacation :D.

i am not sure i understand what isAnonymous gets you that a check for that checking for name does not. so something like `isAnonymous = symbol.valueDeclaration && symbol.valueDeclaration.name;`?